.gitlab-ci-check-golang-lint.yml

```diff
@@ -40,3 +40,18 @@ test:static:
     - fi
   script:
     - golangci-lint run -v
+
+test:govendor-check:
+  stage: test
+  needs: []
+  variables:
+    - GOLANG_VERSION=1.16
+  except:
+    - /^(master|hosted|staging|production|[0-9]+\.[0-9]+\.x|^saas-[a-zA-Z0-9.]+)$/
+  image: golang:${GOLANG_VERSION}
+  script:
+    - if [ -d vendor ]; then
```

Comment:
Are there any repos that have no `vendor` folder? That I can think of, only `integration-test-runner` has no vendor.

I think we can use the opportunity to align all, making this check expect a `vendor` folder and fail otherwise (if so, I think the `go mod vendor` + `git diff` will be sufficient, no need to explicitly check for the folder and fail)

Reply:
and `openssl` and `mender-cli` it seems, and to me aligning due to this seems a little unnecessary tbh, but then again, no hard feelings here.

Reply:
Okey, let's leave it. More flexible.

But `mender-cli` is a core component and I strongly recommend aligning it (can you create a JIRA task for it?).

Reply:
https://tracker.mender.io/browse/QA-357